src/dog.sol

```diff
@@ -66,6 +66,11 @@ contract Dog {
         uint256 chop;  // Liquidation Penalty                                          [wad]
         uint256 hole;  // Max DAI needed to cover debt+fees of active auctions per ilk [rad]
         uint256 dirt;  // Amt DAI needed to cover debt+fees of active auctions per ilk [rad]
+        uint256 trid;  // Amt of dirt dug by auctions.                                 [rad]
+        // Packed in to a single slot to minimze gas costs.
+        uint224 clod;  // Amount of dirt to remove per time quantum                    [rad]
+        uint32  qntm;  // Time between removal of clods                                [seconds]
```

Comment:
how about removing this and setting `clod` in a per-second basis to save some gas?